v3/lints/cabf_ev/lint_ev_valid_time_too_long.go

```diff
@@ -47,7 +47,7 @@ func (l *evValidTooLong) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *evValidTooLong) Execute(c *x509.Certificate) *lint.LintResult {
-	if c.NotBefore.AddDate(0, 27, 0).Before(c.NotAfter) {
+	if util.BeforeOrOn(c.NotBefore.AddDate(0, 27, 0), c.NotAfter) {
```

Comment:
Same comment here.  The lint description says the validity period must be "27 months... or less", a 27 month validity would therefore be acceptable and yet this change makes that an error case.